python-backend/app/api/documents/mines/resources/mine_document_resource.py

```diff
@@ -2,24 +2,34 @@
 import uuid
 
 from flask import request
-from flask_restplus import Resource, reqparse
+from flask_restplus import Resource, reqparse, fields
 from datetime import datetime
+from werkzeug.exceptions import BadRequest, NotFound
 
 from ..models.mine_document import MineDocument
 
 from app.extensions import api
+from app.api.mines.mine.models.mine import Mine
 from ....utils.access_decorators import requires_any_of, VIEW_ALL, MINESPACE_PROPONENT
 from ....utils.resources_mixins import UserMixin, ErrorMixin
 
+MINE_DOCUMENT_MODEL = api.model(
+    'MineDocument', {
+        'mine_document_guid': fields.String,
+        'mine_guid': fields.String,
+        'document_manager_guid': fields.String,
+        'document_name': fields.String,
+        'active_ind': fields.String,
+    })
```

Comment:
Is this intentionally left in this file instead of living in a file of api models?

Reply:
document models will be cleaned up and centralized, It's also possible part 2 doesn't come until reports are done, which will replace all of this code.